virttest/qemu_vm.py

```diff
@@ -1512,11 +1537,13 @@ def sort_key(dev):
                 cmd += "numactl -m %s " % n
 
         # Start constructing devices representation
+        self._probe_capabilities()
```

Comment:
Looks like the right place to put this call is at the beginning of `self.create`

Reply:
ok, will update it. thanks.